Formula/ola.rb

```diff
@@ -20,7 +21,7 @@ class Ola < Formula
   depends_on "libmicrohttpd"
   depends_on "libusb"
   depends_on "numpy"
-  depends_on "protobuf@3.6"
+  depends_on "protobuf@3.14"
```

Comment:
```suggestion
  depends_on "protobuf"
```
We don't need to specify `3.14` since that's already the unversioned formula. Though, I suspect the CI audit will complain about this shortly.

Reply:
Ah, now fixed. I'd just copied what Python was doing. Does this not leave us exposed for next time the protobuf package is bumped?

Reply:
We only keep older `protobuf` versions when necessary - so when 3.15 releases then 3.14 will be gone unless tests show something absolutely needs it. Generally, we try use the latest version if possible. Tests should catch situations where newer versions don't work, so we can decide at the time when protobuf is updated what to do if it does end up breaking this again.

Python is a bit of a unique situation. We pin even the latest version there because we know the next version requires rebuilding _everything_ and doing so to hundreds of formulae in one operation is impossible. It's an odd one out rather than the normal.

Reply:
Thanks for the info both, makes sense.